defaults/README.md

```diff
@@ -77,6 +83,29 @@ __Note:__ If the peer list is empty, your peer may not be registered, or there m
 
 ## Perform [Transactions] via Your Peer
 
+### 0. Prepare Your Client
+
+To interact with your peer, set up a client using one of the following methods:
+
+#### A. SSH into your running peer
+
+#### B. Run a local Docker instance
```

Comment:
The assumption here is that the possible client-peer location patterns are A. remote-remote and B. local-remote. Probably we should support local-local instead of remote-remote